code/modules/mob/living/living_blocking_parrying.dm

```diff
@@ -0,0 +1,139 @@
+// yell at me later for file naming
+// This file contains stuff relating to the new directional blocking and parry system.
+/mob/living
+	var/obj/item/active_block_item
+	var/obj/item/active_parry_item
+	var/parrying = FALSE
+	var/parry_frame = 0
+
+
+GLOBAL_LIST_EMPTY(block_parry_data)
+
+/proc/get_block_parry_data(type_or_id)
+	if(ispath(type_or_id))
+		. = GLOB.block_parry_data["[type_or_id]"]
+		if(!.)
+			. = GLOB.block_parry_data["[type_or_id]"] = new type_or_id
+	else		//text id
+		return GLOB.block_parry_data["[type_or_id]"]
+
+/proc/set_block_parry_data(id, datum/block_parry_data/data)
+	if(ispath(id))
+		CRASH("Path-fetching of block parry data is only to grab static data, do not attempt to modify global caches of paths. Use string IDs.")
+	GLOB.block_parry_data["[id]"] = data
+
+/// Carries data like list data that would be a waste of memory if we initialized the list on every /item as we can cache datums easier.
```

Comment:
isn't this what elements are for,

Reply:
can you imagine flooding SSdcs elements with something like armor datums 

'cause I can't =|

that said a lot of other globals probably belong in subsystems lmao

Reply:
also it wouldn't work well

when I use ID systems it's with the intent of being able to **easily** runtime modify it. 
Yeah you CAN do that with elements but it's like.. why?